fabtests/multinode/src/core_coll.c

```diff
@@ -208,11 +220,18 @@ static int sum_all_reduce_test_run()
 		return err;
 	}
 
-	for (i = 0; i < pm_job.num_ranks; i++) {
-		expect_result += i;
+	for (i = av_set_attr.start_addr;
+	     i <= av_set_attr.end_addr;
+	     i += av_set_attr.stride) {
+		expect_result += i + 1;
+		if (i == pm_job.my_rank)
+			participating = true;
```

Comment:
and do:
```
participating = (i == pm_job.my_rank);
```

Reply:
Fixed using the new is_my_rank_participating() function.